src/crypto/crypto_common.cc

```diff
@@ -297,6 +297,8 @@ const char* X509ErrorCode(long err) {  // NOLINT(runtime/int)
   const char* code = "UNSPECIFIED";
 #define CASE_X509_ERR(CODE) case X509_V_ERR_##CODE: code = #CODE; break;
   switch (err) {
+    // if you modify anything in here, *please* update the respective section in
+    // doc/api/tls.md as well
```

Comment:
I'm not sure this is useful to add a comment here. The chances for this list to be modified is very low – it hasn't changed in the last 8 years, since it was "introduced" in b9a0eb0688104f3619e75e1b9a698080e10d1ec7 (and it's even older than that but you got the point).

```suggestion
```

Reply:
Hm, not sure that I agree with you here. Especially as this code is really old, while it is unlikely that it will be changed ever again, if it will, updating the docs will surely be forgotten. But I'm not insisting on keeping this commit.

Reply:
OK, let's wait and see if anyone wants to share their thoughts on it. One thing to consider is that doc-only commits tends to be backported faster to LTS release lines.

Reply:
Sure, I'm very much open to opinions from contributors as I'm not really familiar with node's development. I'm also not really in a rush to get this backported, so that would be fine by me.

Reply:
The comment doesn't hurt I think :-)